examples/pylab_examples/log_test.py

```diff
@@ -10,10 +10,13 @@
 import matplotlib.pyplot as plt
 import numpy as np
 
+fig, ax = plt.subplots()
+
 dt = 0.01
 t = np.arange(dt, 20.0, dt)
 
-plt.semilogx(t, np.exp(-t/5.0))
-plt.grid(True)
+ax.semilogx(t, np.exp(-t / 5.0))
+ax.grid()
 
 plt.show()
+
```

Comment:
I thought that it was the oppose - that blank lines at the end of a file are required?